### DISCUSSION TITLE
Can anyone explain step by step in detail?

### DISCUSSION THREAD

**Discussion by darkswann** [community/community#19577] [Other Feature Feedback, Questions, & Ideas] [answered]
![Ekran Alıntısı](https://user-images.githubusercontent.com/80572575/176038238-972fa5ce-e421-4484-b92b-02bc35a95bfd.PNG)

**Comment by ertugrul37** [comment, marked as answer]
After answering two discussions, You will get Galaxy Brain badge

More details here:
https://github.com/Schweinepriester/github-profile-achievements

@darkswann
